releasenotes/notes/0.25/add-control-flow-to-consolidate-blocks-e013e28007170377.yaml

```diff
@@ -1,9 +1,4 @@
 ---
-features:
+features_transpiler:
   - |
-    Enabled performing the :class:`.ConsolidateBlocks` pass inside the
-    blocks of :class:`.ControlFlowOp`. This pass collects several sequences of gates
-    and replaces each sequence with the equivalent numeric unitary gate. This new feature enables
-    applying this pass recursively to the blocks in control flow operations. Note that the meaning
-    of "block" in :class:`.ConsolidateBlocks` is unrelated to that in
-    :class:`.ControlFlowOp`.
+    Enabled support for :class:`.ControlFlowOp` operations in the :class:`.ConsolidateBlocks` pass.
```

Comment:
Did you want to say "Previously, the blocks in control flow operations were skipped by this pass." like you did for commutative cancellation?

Reply:
I'm not sure if that's true or not. Do you know off hand? I was just going off the original wording of each.

Reply:
I don't recall I can check the PR. But it's fine like this. I only asked because back to back it was a noticeable difference.